newsroom/oauth_clients/clients_async.py

```diff
@@ -0,0 +1,47 @@
+from content_api import MONGO_PREFIX
+from typing import Optional, Annotated, List, Dict, Union
+from superdesk.core.resources import ResourceModel, ResourceConfig, MongoResourceConfig
+from superdesk.core.resources.service import AsyncResourceService
+from superdesk.core.web import EndpointGroup
+from pydantic import Field
+import logging
+from bson import ObjectId
+
+
+class ClientResource(ResourceModel):
+    id: Annotated[Union[str, ObjectId], Field(alias="_id")] = None
+    name: str
```

Comment:
`name` must be unique, so you will need to add the unique [validator](https://github.com/superdesk/superdesk-core/blob/async/superdesk/core/resources/validators.py#L173) ([example usage](https://github.com/superdesk/superdesk-core/blob/async/tests/core/modules/users/types.py#L39))